From d22e1e6e2a218660aece4b33644c0fcef8cb6baf Mon Sep 17 00:00:00 2001
From: Adam Duskett <Aduskett@gmail.com>
Date: Tue, 5 Feb 2019 10:49:19 -0500
Subject: [PATCH] fix gobject pc file

Newer versions of pkg-config will not respect --variable.and instead will
try to point to the host directory structure.

Instead, add a tools_prefix to the gobject-introspection-1.0.pc.in file
and then sed the location in the .mk file to point to $(STAGING_DIR)/usr/bin

Signed-off-by: Adam Duskett <Aduskett@gmail.com>
---
 gobject-introspection-1.0.pc.in | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/gobject-introspection-1.0.pc.in b/gobject-introspection-1.0.pc.in
index a08b5d2..9a92666 100644
--- a/gobject-introspection-1.0.pc.in
+++ b/gobject-introspection-1.0.pc.in
@@ -1,14 +1,15 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
 bindir=@bindir@
+toolsdir=@bindir@
 datarootdir=@datarootdir@
 datadir=@datadir@
 includedir=@includedir@

-g_ir_scanner=${bindir}/g-ir-scanner
-g_ir_compiler=${bindir}/g-ir-compiler@EXEEXT@
-g_ir_generate=${bindir}/g-ir-generate@EXEEXT@
+g_ir_scanner=${toolsdir}/g-ir-scanner
+g_ir_compiler=${toolsdir}/g-ir-compiler@EXEEXT@
+g_ir_generate=${toolsdir}/g-ir-generate@EXEEXT@
 gidatadir=${datadir}/gobject-introspection-1.0
 girdir=${datadir}/gir-1.0
 typelibdir=${libdir}/girepository-1.0
--
2.20.1
